### DISCUSSION TITLE
Chips (outline + BG transparente)

### DISCUSSION THREAD

**Discussion by annefranca** [Telefonica/mistica-design#1342] [Proposals]
### Goal

Avaliar nova proposta para aplicação do componente Chips para ser adicionado no Mística.

### Context

Pensando em escalabilidade, solicitamos a aplicação do Chips com borda usando opacidade + background transparente. Dessa forma, vamos ter um componente consistente e reutilizável para todas as marcas.

### Impact

Adicionar essa lógica de opacidade e transparência traria facilidade na aplicação e deixaríamos de lado a necessidade de sempre ter que adicionar um novo token de cor para suprir as necessidades de uso.

### Discussion Questions

ND

### Do you have a visual proposal?

https://www.figma.com/file/iQiz75ADXuedhOaxcbweU9/Chips-(outline-%2B-BG-transparente)?type=design&node-id=0%3A1&mode=design&t=Ep2pV9ZGwLWtO2NM-1

**Comment by yceballost** [comment]
Hi Anne,

Reviewing this discussion with the team, we think that what you need here are secondary buttons because what you are showing are actions. (Usar Pix, Depositar, Pagar Boleto are actions)

You can choose if you want to use default size or small
https://tinyurl.com/25bdlso4

**Reply by annefranca** [reply]
Olá @yceballost 

Como alinhado no início da "mistificação", os Chips se apresentam como ações em vários fluxos do Novo App. Vou anexar aqui algumas telas.

@BrunoBiagioniNeto tem alguma consideração?

![47_Cenários_Selfie   SVA](https://github.com/Telefonica/mistica-design/assets/127137234/d8d80fd4-453d-4ea3-8fde-f89ccfc32eb5)
![48_Cenários_Selfie   SVA](https://github.com/Telefonica/mistica-design/assets/127137234/f334142a-d3de-4258-8b44-8840f60beaf6)

**Reply by yceballost** [reply]
Are you aware that we are mixing two very different components?

**Comment by piresrodrigo** [comment]
@yceballost Yayo, I believe that the labels may be creating a sensation of different behaviors between the Assinatura and Vivo Pay chips. But the objective of both are the same, to take the user to another flow. They are navigation shortcuts.

In Assinaturas we can also consider that "enviar internet", "ativar globo play" are actions. And we decided some month ago to treat it like a chip.

I believe that for consistency we should follow the same logic.
Or everything should be treated as a secondary button and we'd come back to the discussion all over again.

**Reply by yceballost** [reply]
should we remove the current logic of the chip changing in the "alternative" variant and simply make it transparent? I think this would be more scalable considering the use you want to make of it.

https://mistica-web.vercel.app/?path=/story/utilities-themevariant--components-over-different-theme-variants&args=themeVariant:alternative

**Reply by annefranca** [reply]
@yceballost estamos aplicando opacidade da borda. Seria um impeditivo? Acredito que vamos usar a mesma lógica de aplicação do Snap Card, certo? https://github.com/Telefonica/mistica-design/discussions/1343

**Comment by yceballost** [comment]
Chips are also used for single and multiple selections, how would these selections behave with the "transparent" variant?

**Reply by annefranca** [reply]
@aweell @AnaMontes11 
Segue a proposta atualizada: [Playroom](https://mistica-web.vercel.app/playroom#?code=N4Igxg9gJgpiBcIA8AFATjAbgSxgdwBUIIAbAZwAIB6APgB0A7RpAIQgA8KAHAQyimwMA5gF5gAJgAMAX3oMKFJAWwAXEjHE10ELvArYyBNDwZleGBiqRVlajXOtt2DgEowzEU9kwwAMjwBPCABXFX0yAEkGHzQyGApMHjRsExUROhBYADMeYJIVDIosvJIAdWwoFQALOQVWDm4%2BAWEATTEARgA2WUYFOoBxZKgKNAg8Mg7pCkgSYIBbUzEAFimhHi4xKR75PoGhiJUYOenSeYYAZV4GZamAK2CyFWwsgPOYEiz08BhLGDQM2q7PpKGDsFRLGgEKpHeIANSSKUsegAIjAcnkrDZQeDAbtrIMKgcjrjgQSoETjjwSNghBd3p8MmAfod-iAKPdHs9XvSvkzfqySXjzioeGAANYUMyimBiGSCoEgsHtGjCniHFFo3L5awEbHK3pAupUVXi%2BV1KLUhjxKVMr5JEIMKAAg2GuomiU2mXAAAc21dCotghgZtdSAEmBd-qBjwC6jEwEjUaBACMIGhYGgXHxsA89GQxYJ4bEAHSp9N-LMCbDFwQCMBqtMAGkTSb6ZYzegy7S4nDIpAqGWbO1bKbTGYAwqQ03oZmmyKWx38hyPdtJpC3-SH-fVOLx%2BIJRMB2uIpnvmkIABpfLrO4cruq6sGacdVbBcMg67GaDdRxwcLcKlQ4YAUagZWgB1juiG1hgcGLb4vshxzPKSBkhSFBUjSdIfLyzJ-IUHJPC8bw4YyeECj%2BSDupKvC2sAco-g%2BeoqiK6oUAAgmATyJIcUCfkq0HGiKpqUdRnpiL6EGwSBijhox0YqLGXoJneI7thW2a5pKBYMEW87qZm2bVrW2D1ioTbyYaBmdiA3a9v2TogMu94UAZk4kNOJweSWkCWKMJCcdxaowFAzn3smopikIozBI67mebOJbJsYjq%2BGMYWtmulkUDJdROI0%2B7CB0J4FeeV5dp0t4uYoj4qM%2Br7vvxdW5YoVBODJ1jAfJMEMJacGqcCQmRShVBoUhKFjUcGHUrSJEMt8-IEQ8RHcqRC0slVoZibRXoMQNeK1cqqpsai6LaliAmUUNIn7Yo%2BVngebRHieEHbdKyx7SOYbeNlCgxnGwAqdVBmVjmZB5jpekLuWhlVsWvkikGaAZWpi5oDZdmSg5g6-W2aPxejXlztDGYo1GWW3bsLU7qVB4dN0LVusJHo7fTfrVYoES%2BQAsp4MABMioxcG4Qh5Ek1CM8Cb10Z9HNMU%2BNDjkkwynVqmK1d%2BlOtoqdVE2gYiJfOhxgm8vlQEkASyKiZBgMkXBPJ4TWa3LkHM5LrvDbjf7ONlnXeBB11ioJ7XwaNiHEvBk2UjN2HzXyG1soRXJzbhi0gNB0u7ez26HSxwV6IF3jBXxF0qPqt0ezd273U0j3FdnW3MzR73AEsstJt9EZa7s-3KbjCgg5p4PaYWST6WjoPVgjPBI2TSbWRQXY9lj1KOXPUZuVOhOG-DngqP5hc8SF6-%2BhF4rRQ6UAEzOW-6SlUBpXgJ9AhTI7UzXhWHl0DdfZnbOS%2BaHmfMBZCxFmLNAEt%2B5SybuJeiP8XI62fMrDiQUoAQCdgA4EGs9YG1vsWY2KhTaeHNmgS2NBra2zfA7BgGCoFGiglAyuQcvZtX-L7IC-tuqB2DmwgaCFCTjUjPwqAgJvYOCoG4DwXgfD%2BCCKEVw7guCeDIN4PwgQQhhESMkVIdp8h-AYGqVRhRigkDKBUaooiP7niet-EkqEhgjDGBMI8UwZhnGcSsCgawNgSHbqScOxw3ELEuCYG47JlrJx5GRNO0FaoQihDCCgRZEQqALnotABjuIwCahCUOUcJoBOmlhFO0SE7hM5MRKJ618Lp1EjA1mcCIK52OjADUZ11bMSugwiusFm62gyPaWKjlXr1JbpJX20kvZyW7gpJS8Y6GuQnkPCGo8kpLLhiZMyFkZlWTRhjZefZV44x2aOGG189bjxhs-Q0r8VzuysXTZ6p5a7CHKrZSqtSTkHS-IrBqH5S7OwQawn2Jy-aYA6lQSZWsmGCShYaYRFICkCKmphWaVT441PKStEp1SKIVz-o07qzTWKtI4lxIuvEmrl23NwupkU%2BlenGdCuF2tpn3l7vMr5iyYagy0vmVZlyMyTxrI6UyDZkZQIXkveyRynKSvxlvG%2B3l5wIwPuSo%2BoVJWRQvkM85O9komAfulXGtykzvwaA9IqTzaavOvB892Gtflvn%2BY6lhIcZlgoDiy%2BFtK%2BFh2RchSOhTUWx1TmUpOlS1oYrxduAlfj4XEvzhQVWGIqWCW6dXC1LyhA2Jet1Albd4HwrZSuDlgN5U8uWSPXSY8SYaThtPWeFaOyL1sgc7GcquWb28kq4mBlrl9FNVGc1u5s3-0YQS2xCykBc08LzK0ICdBgJIOLWg0641FpXIgxWyCU3nVdVy75T4cHAB3gQohjoLZW3cJQ%2B22BHYAswTCxhvr7xiPYV1aFr7QzApGvkoNAailoqjeRJaFTVpx1A582NoyZabvlmXPObFD7FzTV0t2lEHmtHriM%2BlsDC0dRLSOMtQMXKDwEHyyGtbyPYCnnvGeVoJVdr2a2zGhyByduBgqntFzd5%2BVICh3iA7dhnyijFOKireMGtSsak5Q7Nxeyw1-BmE7YNeinYezmQCF2CyXTAUWK6IFrs01RNTsp4Pa0dUrdMKCi5oNoZphD4gT1nuxBekhZCKF22oQ5l23633%2Be1r%2Bj9nCv0Zp-e6n1-67zCNEb%2BhMbIMjrhoRIxRyjVGyI0QoqRKiZHqNCOEKIMQ4gJARDojIghiswGMSUcolQagGhppanN46YtkkceMSYXl3FhO8ZseNigo7deCVcMJEaINhpqbE7E8ToRzDhGVpEFAit-DiDkv9ASkXkiQkB0NpTMXjZxdGza8KN1NOYi0tpas0MV0C3iXp4kBnieGfmszPoLPml6kGamRHS2KQBqRrjlaKPD35TWtZwPaMirrOK4TpyW3SpXhxuHInuOeXWFwEgAR2IkC4FUHgAAKMHUNjB4AJoKpcFBJDFk6AAShRxQeTw7GFKfrjay8dqTsIKs383zQLItbo4eC9h3qf3hZ9aLo00X4VDZDUdqDWLIkgZiXS8UDLzPnYEpdslQVKWl2pWLjD%2BK3tMo7pL0Mv2kwkebfWsGKzwcU9hlDzZsOtXn2e3qvBomdUScsAxv4LB74AHkYgrdiDAOHTPDQjvZ2z5rbybzQe5z8l8zq%2BdfWCx6oXXqvvgXQ57P10uFSy5jvLtOivI2QZV8bvDDSBtHsQ9rvdHTLq3fFwqJTuaLOmdry3AjvtLdRmt8xyHlGBV1qd3Rv3TaR8I7bTK5HNu0DnIx1jnHePCfE9raT8nE-GxU5p-Tk1yW36KazZ-Vr1Ue9q9gRpl2s6GDzv5rp4W%2BnwGQJM2dhZTmd02eb%2BnjmRBFzPBc9GAM2K9chG9bze9FLA9PzdvDPBA38TPDPT9DuO7QaAXPEf1bbCOPhEvYpdFBXQ7Ig6vGDXvODTXRvElAudVVDfXdNI3TNUdT%2BLvXDG-BpfvD1Qff0YfIHIVKtLfCHAQhtejGffgv4fZBfNeJfT3ZVPjfeATOgoTN3MTS%2BOQvtbVD3MQxjQPQ1dAdwOITVOTE-VsGPZrS-BBSdFTEzB-J-RdV-AzVdJ9L-RzGqFPZBdiVBdBR9b-dw49RKXBeQ0A8A0ha9G2aAh9OAwAjA9ApAmlLA5AtA5A%2BI1qRIqXTbIRHAuLdqBLVtZLawSRJRaRNRORFQOQZgIo9LPLMo0rbRSwL4SrVbarNkExMxerVyLQ9QreMQVfbHXHfHInKjEsHfL3e%2BffanAATlp22CEU70sLxHa1GE6xcWG0WFblWHWH603XsUAyCQuFGw2Irwm32xjV-DiUhDmwW3qNSWW2iGaIoAAGoKAWAuihkOJ0lMlVF1sroi8Zdg1S9SDw0IlK9JszjyCOCW568FRE0TpNRU0GCuFUjPs%2Bp1cntL4ucISWYxkPtgRzcO4eDWw%2BC5ZuURC7dq0oYaNjJRUtkmMSTSTJDWN21ZUGdUczlJM%2Bj19BihD5xRj5D%2B0D86dWTGdTD7wn1WdrV49Ock8XYec09fCTMUCgVkiM98TAJkTWo1SEVBEK4CDgMq9gTwMy8E52DsTKDfZYTSVBMQobs4imDfwCVTdtY1Ti0fouViSJCncx8HcJ9hUXdzI6SOZvdtCeN9U3jfdEZdDg9Q97jw8Gco9XR7lz9zw49s0E97Vp05TGoFSr8lTUDQtVTc9%2Bo7SC8aU-ji8ATCDldDTsUgSptXsKCs4OpLSrsETc4kT7TmD2c2DfYC1oTXSu5qoPSyN1lySeTfSjIFD-dAzPSpCkcZDNNu10cuBMd%2BiN8hjx8%2BS%2B00YJjD94zRSzCWdkzHk79ADrDcTtw7DgEX9l1nC-Dr8zSmy-Cf9rMVZ4T90flnz-DdZAjT0QC3MwDiEICvMqEYCADcyNSaVILAJ0ikiCygsNT31btyzFjKz9SwSwNazqz6zoVXCvoWydcKUbTESwtOzQx5inkOo%2ByLzZI3Shz-s%2B5FzRzvTKTRyp9IylwFlgzujQyvdwyr4dCA974DCyAjD9zcYkyWCUyjwbCXZzzMFAE51rzQE39DMP96SHz1dCU3CXzPDvDwLADsFfzXMTZALL0wjICIjQKojPy3Dn1FToLsDYKaUVSUiyKfVnLWoUKvKhg7FM9Ci0sSjMt5FGAnIQBqgYQyAEAABtEAWEbwCAAAfQADl8AQAABdRsEAPAcxKoKK%2BAaKgAdk6G9HSukCAA)

**Reply by aweell** [reply]
I share here the team proposal to cover:

* All chips no matter the theme context / color scheme have transparent background
* Partially solved active state (for this we'll need to create a new token `controlActivatedInverse` that is also needed for other controls that actually are not working perfectly in inverse contexts)

https://tinyurl.com/2kfg9jo3

**Reply by annefranca** [reply]
@aweell
Obrigada pela sua proposta, mas temos uma nova proposta para que você possa analizar que seria escalável para todas as marcas que consomem o Mística.

Links com a proposta atualizada: 

https://www.figma.com/file/iQiz75ADXuedhOaxcbweU9/Chips-(outline-%2B-BG-transparente)?type=design&node-id=1639%3A54797&mode=design&t=XnEJUjDKxahQ0acu-1

https://tinyl.io/9381

**Comment by yceballost** [comment]
This component won't be part of Mística by the moment, will be developed in a Vivo extended library

**Reply by annefranca** [reply]
Obrigada, @yceballost 

Vamos iniciar o desenvolvimento em Extended.
